src/Microsoft.TemplateEngine.Edge/Settings/TemplateCache.cs

```diff
@@ -45,10 +45,15 @@ public TemplateCache(IEnumerable<ScanResult> scanResults, Dictionary<string, Dat
                     // This localization is either the perfect match, or a suitable substitude and there are no other candidates for the job yet.
                     localizationsByTemplateId[locator.Identity] = locator;
                 }
+
```

Comment:
it would be good to add comment here why it's needed.

Reply:
I added comment where dictionary is defined, I hope that is OK...